src/Nix/Builtins.hs

```diff
@@ -62,6 +64,7 @@ import           Nix.Expr.Types
 import           Nix.Expr.Types.Annotated
 import           Nix.Frames
 import           Nix.Normal
+import qualified Nix.Options as NOpts
```

Comment:
I don't use qualified names too much when the module doesn't contain overriding definitions. So you'll see it on HashMap, for example, but rarely on our own modules.

Reply:
@jwiegley Without importing qualified I get the following error: 
```
src/Nix/Builtins.hs:128:50: error:
    Ambiguous occurrence ‘currentTime’
    It could refer to either ‘Nix.Options.currentTime’,
                             imported from ‘Nix.Options’ at src/Nix/Builtins.hs:67:1-28
                             (and originally defined at src/Nix/Options.hs:42:7-17)
                          or ‘Nix.Builtins.currentTime’,
                             defined at src/Nix/Builtins.hs:906:1
```
Would you like me to rearrange some naming conventions?